src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="search-form big-search-form"
+    >
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-input search-input-big"
```

Comment:
please follow approach form [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p2

<img width="1033" alt="Screenshot 2023-03-21 at 19 37 30" src="https://user-images.githubusercontent.com/32853034/226709014-021fecf8-e8fb-4d5d-8f9a-4c7baa6d9480.png">

```suggestion
        class="search-input"
```